src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -230,6 +230,17 @@ public class DurableTaskExtension :
         /// </value>
         public int ExtendedSessionIdleTimeoutInSeconds { get; set; } = 30;
 
+        /// <summary>
+        /// Gets or sets if logs for replay events need to be recorded.
+        /// </summary>
+        /// <remarks>
+        /// The default value is false that prevents logging the replay events.

```

Comment:
"The default value is false , which disables the logging of replay events."

Reply:
Thank you! I will fix it.